scaffold/bitbucket/bitbucket-pipelines.example.yml

```diff
@@ -0,0 +1,156 @@
+image: tugboatqa/php:8.0-apache
```

Comment:
Is this intended to be "copy and never update" or "somehow get updates"? I wonder if it makes sense to rebase on top of https://github.com/Lullabot/drainpipe/pull/123/files#diff-fccaa14362cd8508cb094a15287d2e442fc57d94205c6b2146087a552815d1fcR337 so you can dynamically get the PHP version and set it via twig.